sdcm/nemesis.py

```diff
@@ -4372,3 +4416,16 @@ class StartStopValidationCompaction(Nemesis):
 
     def disrupt(self):
         self.disrupt_start_stop_validation_compaction()
+
+
+class EnableAuditingDmlTable(Nemesis):
+    """
+    class for auditing nemesis. The feature is available only on enterprise versions.
+    Auditing configuration-
+    audit: "table"
+    audit_categories: "DML"
+    """
+    disruptive = False
```

Comment:
because you run `node.restart_scylla_server` it should be considered as `disruptive = True`